server_admin/topics/clients/saml/proc-creating-saml-client.adoc

```diff
@@ -0,0 +1,112 @@
+[id="proc-creating-saml-client_{context}"]
+
+=== SAML Clients
+
+{project_name} supports <<_saml,SAML 2.0>> for registered applications.
+POST and Redirect bindings are supported. You can choose to require client signature validation. You can have the server sign and/or encrypt responses as well.
+
+.Procedure
+. Click `Clients` in the left navigation pane.  
+
+. Click *Create* to go to the `Add Client` page.
++
+.Add Client
+image:{project_images}/add-client-saml.png[]
+
+. Enter the `Client ID` of the client. This is often a URL and is the expected `issuer` value in SAML requests sent by the application.
+
+. Select `saml` in the `Client Protocol` drop down box.
+
+. Enter the `Client SAML Endpoint` URL. This URL is where you want the {project_name} server to send SAML requests and responses. Generally, applications have one URL for processing SAML requests. Multiple URLs can be set in the `Settings` tab of the client.
+
+. Click *Save*.  This action creates the client and brings you to the `Settings` tab.
++
+.Client Settings
+image:{project_images}/client-settings-saml.png[]
++
+The following list describes each setting:
++
+`Client ID`:: The alpha-numeric ID string that is used in OIDC requests and in the {project_name} database to identify the client. This value must match the issuer value sent with AuthNRequests. {project_name} pulls the issuer from the Authn SAML request and match it to a client by this value.
+
+`Name`:: The name for the client in {project_name} UI screen. To localize
+the name, set up a replacement string value. For example, a string value such as $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}] for more information.
+
+`Description`:: The description of the client.  This setting can also be localized.
+
+`Enabled`:: When set to OFF, the client cannot request authentication.
+
+`Consent Required`:: When set to ON, users see a consent page that they can use to grant access to that application.  It will also display metadata so the user knows the exact information that the client can access.
```

Comment:
@briandooley Did you remove this because it is for GooglePlus which no longer exists?